python/cuml/cuml/internals/logger.pyx

```diff
@@ -44,6 +44,21 @@ IF GPUBUILD == 1:
             sys.stdout.flush()
 
 
+def _verbose_to_level(verbose: bool | int) -> level_enum:
+    """Parse the common `verbose` parameter into a `level_enum`."""
+    if verbose is True:
+        return level_enum.debug
+    elif verbose is False:
+        return level_enum.info
+    else:
+        return level_enum(6 - verbose)
+
+
+def _verbose_from_level(level: level_enum) -> int:
```

Comment:
Ideally these would be methods on the `level_enum` enum (e.g. `level_enum.from_verbose(verbose)`...), but cython doesn't seem to like any of the ways I wanted to spell this. Leaving these as internal utils for now.